saleor/payment/models.py

```diff
@@ -149,16 +149,19 @@ def can_capture(self):
 
     def can_charge(self):
         not_fully_charged = (
-            self.charge_status == ChargeStatus.CHARGED
-            and self.get_total() > self.get_captured_amount())
+            self.charge_status == ChargeStatus.PARTIALLY_CHARGED)
         return self.is_active and (self.not_charged or not_fully_charged)
 
     def can_void(self):
         return self.is_active and self.not_charged and self.is_authorized
 
     def can_refund(self):
+        can_refund_charge_status = (
```

Comment:
We could make it constant in the class (`self.CAN_REFUND_CHARGE_STATUS`). I'm not sure Python  make those local variables static.

Reply:
After a second thought, I decide not to refactor it since ```can_refund_charge_status``` is not used somewhere else. Besids, even it is defined as ```CAN_REFUND_CHARGE_STATUS```, there is no way to prevent changing it since there is actually no constant (no ```const``` as other language) in Python.